### PR TITLE
Teshari except they're awesome because they're Avali

### PR DESCRIPTION
**Pull request by NullDagaf**
## About The Pull Request

adds avali and shitmafucks the code massively

## Why It's Good For The Game
no lol

## Changelog

:cl:
add: adds the chicken race
/:cl:

### CODE REVIEW

**Comment by ChungusGamer666** [comment]
this pr singlehandedly destroys code quality by 3 nanogazillion so awesome

**Comment by ChungusGamer666** [comment]
most of everything seems working

**Comment by ChungusGamer666** [comment]
pr is good and ready to be merged except for markings as we discussed on discord, i want to get this in ASAP so further improvements can be more easily made

i hate teshari btw

File: code/__DEFINES/species_clothing_paths.dm [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-#define DEFAULT_SHOES_FILE 'icons/mob/clothing/feet.dmi'
-///The dmi for humanoid oversuits
-#define DEFAULT_SUIT_FILE 'icons/mob/clothing/suits/default.dmi'
-
 //MONKEY PATHS
 ///The dmi for monkey uniforms
 #define MONKEY_UNIFORM_FILE 'icons/mob/species/monkey/uniform.dmi'
 
 //DIGITIGRADE PATHS
-///The dmi containing digitigrade uniforms
```
Comment (ChungusGamer666) [inline code/__DEFINES/species_clothing_paths.dm:14]: why are you removing these comments and the defines are staying?

Comment (NullDagaf) [reply code/__DEFINES/species_clothing_paths.dm:14]: i remove when im in home

File: code/modules/clothing/under/syndicate.dm [hunk 2]
```diff
@@ -69,7 +70,7 @@
 /obj/item/clothing/under/syndicate/tacticool/examine(mob/user)
 	. = ..()
 	. += "It has a label that says cleaning this 'genuine' Waffle Corp. product with cleaning solutions other than Grime Liberator telelocational podcrystals will void the warranty."
-	. += "What on earth is a <font color='red'>tele</font>locational pod<font color='red'>crystal</font>?"
```
[the remark this answers is not in the document]
Comment (ChungusGamer666) [reply code/modules/clothing/under/syndicate.dm:72]: soul is still gone....

**Comment by ChungusGamer666** [comment]
loathesome birds

**Comment by ChungusGamer666** [comment]
will try and go back to this and make some improvements later, right now this is ok i just want this fullmerged already by god...

**Pull request merged**